pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -1,5 +1,5 @@
-{ stdenv, lib, fetchFromGitHub, fetchpatch, pkgconfig, intltool, gperf, libcap, kmod
-, xz, pam, acl, libuuid, m4, utillinux, libffi
+{ stdenv, lib, fetchFromGitHub, fetchpatch, pkgconfig, intltool, gperf, libcap
+, curl, dbus, kmod, gnupg, gnutar, xz, pam, acl, libuuid, m4, utillinux, libffi
```

Comment:
`dbus` doesn't seem to be referenced anywhere in the derivation.

Reply:
Will adjust soon, thank you for the notice!